modules/virt-routes-route-rules.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-routes-route-rules_{context}"]
+= Routes and route rules
+
+[role="_abstract"]
+After you configure an IP address for a network interface, you can configure routes and route rules in the NMState configuration for cluster nodes. 
+
+[IMPORTANT]
+====
+You cannot use the OVN-Kubernetes `br-ex` bridge as the next hop interface when configuring a static route unless you manually configured a customized `br-ex` bridge.
+
+For more information, see "Creating a manifest object that includes a customized br-ex bridge" in the _Deploying installer-provisioned clusters on bare metal_ document or the _Installing a user-provisioned cluster on bare metal_ document.
+====
+
+The `routes` parameter defines static routes and these routes determine the traffic that leaves the network interfaces and the destination network for the traffic. You can specify the following values for the `routes` parameter:
+
+* `running`: Directs the NMState Operator to load the running, active, network configuration routes on the host node.
+* `config`: Directs the NMState Operator to add only specific routes to host nodes. Depending on the NMState enforcement policy, the Operator ignores or removes unmanaged routes that exist on cluster nodes.
+
+[NOTE]
+====
+After you apply an NMState configuration to cluster nodes and you want to change existing routes, you must specify the the old route in the `state: absent` parameter and the new route in the `state: present` parameter. The NMState Operator can then delete the old route and apply the new route to cluster nodes.
```

Comment:
> you must specify the the old route in the `state: absent` parameter

I would replace "in" with "with". "state: absent" isn't a section that contains other things, it's just a parameter on a larger configuration.

Reply:
:heavy_check_mark: